src/components/field/_field.scss

```diff
@@ -17,6 +17,7 @@
 
   &__group {
     @extend .u-cf;
+    margin-bottom: -1rem;
```

Comment:
The clearfix above already seems to being doing this job?  There are now spacing issues caused by this negative margin...
<img width="1278" alt="screenshot 2019-02-21 at 14 31 35" src="https://user-images.githubusercontent.com/43346934/53176353-ccb97580-35e5-11e9-8d6e-77efb16d3337.png">

Reply:
Hmm the clearfix didn't appear to have the effect of preventing the bottom margin on the date input. I'll take a deeper look into it.

Reply:
Having had a look this behaviour is correct. Currently field doesn't actually apply any spacing between itself and another field (see https://sdc-prototypes.netlify.com/prototypes/example/ for an example). In runner this is achieved by wrapping each field in a custom `answer` class. 

We should maybe discuss the this margin being applied to field themselves outside of this PR.

Reply:
I've began a discussion about this on the forum issue for field: https://github.com/ONSdigital/pattern-library-v2/issues/35#issuecomment-466045681